s2p/initialization.py

```diff
@@ -128,10 +129,16 @@ def build_cfg(user_cfg):
             if d in cfg['images'][i] and cfg['images'][i][d] is not None and not os.path.isabs(cfg['images'][i][d]):
                 cfg['images'][i][d] = os.path.abspath(cfg['images'][i][d])
 
-    # get utm zone
-    if 'utm_zone' not in cfg or cfg['utm_zone'] is None:
+    # get out_crs
+    if 'out_crs' not in cfg or cfg['out_crs'] is None:
         x, y, w, h = [cfg['roi'][k] for k in ['x', 'y', 'w', 'h']]
-        cfg['utm_zone'] = rpc_utils.utm_zone(cfg['images'][0]['rpcm'], x, y, w, h)
+        utm_zone = rpc_utils.utm_zone(cfg['images'][0]['rpcm'], x, y, w, h)
+        epsg_code = geographiclib.epsg_code_from_utm_zone(utm_zone)
+        cfg['out_crs'] = "epsg:{}".format(epsg_code)
+    try:
+        geographiclib.pyproj_crs(cfg['out_crs'])
+    except pyproj.exceptions.CRSError as e:
```

Comment:
`pyproj` is not imported in this module, so if this line is reached it raises an `ImportError`

Reply:
You're right. I removed the `pyproj` import after implementing a wrapper around `pyproj.crs.CRS` in `geographiclib.py`.